edtf/views.py

```diff
@@ -40,6 +40,19 @@ def result_json(request):
         jsonedtfdict = {
             'validEDTF': valid_edtf.is_valid(date) and funcs[level]
         }
+    elif 'levelConforms' in request.GET:
+        levelConforms = request.GET.get('levelConforms')
+        if levelConforms == '' or levelConforms is None or int(levelConforms) > 2 or \
+                int(levelConforms) < 0:
```

Comment:
Since we only have three valid levels, what do you think about setting a constant of `LEVELS = ('0', '1', '2')` near the top of this file and using it for checking `level` and `levelConforms` in the conditional with something like:
```
if `level` not in LEVELS
```
and
```
if levelConforms not in LEVELS:
```